docs/api_reference/python-api-reference.md

```diff
@@ -65,6 +84,8 @@ SPDX-License-Identifier: MPL-2.0
 ```{eval-rst}
 .. autofunction:: power_grid_model.utils.get_dataset_type
 .. autofunction:: power_grid_model.utils.get_dataset_scenario
+.. autofunction:: power_grid_model.utils.get_dataset_batch_size
```

Comment:
Should we also expose here the `compatibility_convert_row_columnar_dataset` function? It has proven to be really valuable for us, I bet it would be for the user.

Reply:
then we should come up with a better name. any suggestions?

Reply:
Definetely needs a better name, but also probably needs to be discussed on the knowledge sharing (?). As for names, let me think, nothing pops to mind at the moment. Maybe we can just scope it out for know and tell Peter to add it to some agenda to discuss it with the team.

Reply:
let's discuss later and scope out for now idd